### DISCUSSION TITLE
LTE/4G module instead of LoRa

### DISCUSSION THREAD

**Discussion by fantoelec** [nzymedefense/nzyme#424] [Questions] [answered]
hi, 
Is it possible to build the trackers with raspi4 and LTE/4G Hats instead of using the LoRa module..?

Kind regards
fantoelec

**Comment by lennartkoopmann** [comment, marked as answer]
Not yet, but additional protocols will be implemented in the future, with one of them being a simple TCP/IP implementation that would allow you to use LTE, WiFi or other IP networks.

**Reply by fantoelec** [reply]
Thanks !
